promptify/parser/parser.py

```diff
@@ -234,14 +234,14 @@ def fit(self, json_str: str, max_completion_length: int = 5) -> Dict[str, Any]:
                 )
                 return {
                     "status": "completed",
-                    "object_type": type(output["completion"]),
-                    "data": output,
+                    "object_type": type(output),
+                    "data": {"completion": output, "suggestions": []},
                 }
             except Exception as e:
                 return {
                     "status": "failed",
                     "object_type": None,
-                    "data": {"error_message": str(e)},
```

Comment:
Using `'error_message'` without `'completion'` causes error on [this later line](https://github.com/promptslab/Promptify/blob/main/promptify/prompter/nlp_prompter.py#L223).
I just changed it to `'completion'`, since we already have status to indicate error. Alternatively, we can leave it as `'error_message'` but have to change the aforementioned line of code.